<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PC0432J4-T440P" name="DisputeEndtoEndFlow_DisputeTab.AdminSentInquirytoPayee" tests="1" failures="1" timestamp="8 Mar 2018 11:03:17 GMT" time="77.044" errors="0">
  <testcase name="Admin_InquirytoManager" time="77.044" classname="DisputeEndtoEndFlow_DisputeTab.AdminSentInquirytoPayee">
    <failure type="java.lang.AssertionError" message="Failed TestCaseAdmin_InquirytoManager">
      <![CDATA[java.lang.AssertionError: Failed TestCaseAdmin_InquirytoManager
	at org.testng.Assert.fail(Assert.java:94)
	at com.calliduscloud.testing.util.SeleniumMethods.catch_code(SeleniumMethods.java:553)
	at DisputeEndtoEndFlow_DisputeTab.AdminSentInquirytoPayee.Admin_InquirytoManager(AdminSentInquirytoPayee.java:138)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- Admin_InquirytoManager -->
</testsuite> <!-- DisputeEndtoEndFlow_DisputeTab.AdminSentInquirytoPayee -->
